### PR TITLE
adding complete as an alias to finished_success and analysis_completed to grafeas

### PR DESCRIPTION
**Pull request by nms2140**

### CODE REVIEW

File: proto/v1/discovery.proto [hunk 1]
```diff
@@ -52,24 +52,39 @@ message DiscoveryOccurrence {
...
+  // analysis can be performed on a single resource.
+  message AnalysisCompleted {
+    repeated string analysis_type = 1;
+  }
+
+  AnalysisCompleted analysis_completed = 7;
+
+  // Indicates any errors encountered during analysis of a resource. There
+  // could be 0 or more of these errors.
+  repeated google.rpc.Status analysis_error = 8;
```
Comment (nyc) [inline proto/v1/discovery.proto:86]: Should this be a message with the analysis type included as well so we know which analysis failed?

Comment (nms2140) [reply proto/v1/discovery.proto:86]: the Status proto (https://github.com/googleapis/googleapis/blob/master/google/rpc/status.proto) has a details field which will allow us to populate information regarding which analysis failed into it.

**Comment by nyc** [comment]
> > Could you regenerate the Swagger + Go as well?
> 
> I ran go generate, but there were no files / updates generated afaict. Any instructions on how to do this?

You should run `make generate` from the top-level of the repository.

**Comment by nms2140** [comment]
> 
I was able to generate correct for v1beta1. Nothing seems to get generated for v1.

**Comment by nms2140** [comment]
> We should also update v1beta1

Done.

**Comment by nyc** [comment]
Very strange that v1 is not re-generating. Digging into it a bit.

**Comment by nyc** [comment]
It looks like it's because the PR is based on an old version of the code and I made some changes to the generation. Could you merge `origin/master` (or whatever you named the upstream repo) back into your branch then try running `make generate` again?

**Comment by nyc** [comment]
Just for reference, I checked out your PR and ran:

```
$ git merge upstream/master

...

$ make generate 

...

$ git status                                            nms2140/master!
On branch nms2140/master
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   cel/syntax.pb.go
	modified:   proto/v1/grafeas_go_proto/discovery.pb.go
	modified:   proto/v1/swagger/grafeas.swagger.json
	modified:   proto/v1beta1/attestation_go_proto/attestation.pb.go
	modified:   proto/v1beta1/common_go_proto/common.pb.go
	modified:   proto/v1beta1/grafeas_go_proto/grafeas.pb.go
	modified:   proto/v1beta1/image_go_proto/image.pb.go
	modified:   proto/v1beta1/intoto_go_proto/intoto.pb.go
	modified:   proto/v1beta1/source_go_proto/source.pb.go
	modified:   proto/v1beta1/spdx_go_proto/spdx.pb.go
	modified:   proto/v1beta1/swagger/merged/grafeas.swagger.json
```

You can see the `v1` files got generated properly.

**Comment by nms2140** [comment]
> 

I was able to generate the v1 swagger file. Thank you.

**Review by nyc — APPROVED** [review]

**Pull request merged**